Source/Modules/ModuleScene.h

```diff
@@ -33,9 +33,8 @@ class ModuleScene : public Module
 	bool hasNewUID(UID oldUID, UID& newUID);
 	void SetSceneRootAnimObjects(std::vector<GameObject*> gameObjects);
 
-	void SaveSceneToJson(const std::string& name);
-	void LoadSceneFromJson(const std::string& name);
-	void ImportFromJson(const std::string& name);
+	void SaveScene(const std::string& name);
+	void LoadScene(const std::string& name, bool mantainActualScene);
```

Comment:
Maybe we could default the bool to false? Since I imagine is the most common use case for this method.
```suggestion
	void LoadScene(const std::string& name, bool mantainActualScene = false);
```
Just being picky though.